app/src/main/java/com/globant/counter/android/mvp/presenter/CalculatePresenter.java

```diff
@@ -0,0 +1,66 @@
+package com.globant.counter.android.mvp.presenter;
+
+import com.globant.counter.android.R;
+import com.globant.counter.android.mvp.model.CalculateModel;
+import com.globant.counter.android.mvp.view.CalculateView;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author s.ruiz
+ */
+
+public class CalculatePresenter {
+    private CalculateView view;
+    private CalculateModel model;
+
+    public CalculatePresenter(CalculateView view) {
+        this.view = view;
+        model = new CalculateModel();
```

Comment:
The model object is passed in the constructor as parameter. Because, you will not be able to mock the model in the presenter test.